src/components/HFUI/HFUI.js

```diff
@@ -117,13 +99,9 @@ const HFUI = (props) => {
 
   useEffect(() => {
     if (authToken) {
-      getSettings(authToken)
-      getFeatureFlags(authToken)
-      getFavoritePairs(authToken, currentMode)
-      getPastStrategies(authToken)
```

Comment:
Mostly for hosted version removing this won't fetch these data, as auth token is coming from url and not through `info.auth_token` event. 
I need to check this practically but staging/prod hosted version is not working, could you check in code or locally if hosted version sends this `info.auth_token` event. If not we have to keep this api calls here.

Reply:
Great catch! Hosted is already outdated and incompatible with the recent dependencies versions, and we will have to work on a lot of stuff before going live again. Anyways, I took note of this on [ASANA](https://app.asana.com/0/1201848935859401/1202560377040002/f)

Reply:
mostly all things on FE side works for hosted, so lets keep calls as it is on HFUI so it's compatible and we don't break what's already working

Reply:
also those existing calls will only be re-fetched if authToken changes, so it won't have performance impact as I understand

Reply:
past strategies must be refreshed when switching modes, but I can add that event there instead of removing it.

Reply:
aah, `useEffect` is for that purpose only that it don't refetch data without dependency change (authToke in this case), but something else is causing this. will look into this, thanks for sharing.

Reply:
fixed in https://github.com/bitfinexcom/bfx-hf-ui-core/pull/460